aws-samples/pcluster-manager
# Apply linting and formatting to the frontend codebase

## Description

With [this PR](https://github.com/aws-samples/pcluster-manager/pull/287) we introduced linting and formatting through `ESlint` and `Prettier`. This PR aims at applying linting and formatting rules to the whole frontend codebase, as a starting point for having a uniform style across all JS/TS files.

## Changes

* Applied linting and formatting to the frontend codebase following rules specified in `.prettierc.json` and `.eslintrc.json`.

## Changelog entry

* N/A



## How Has This Been Tested?

* PCM built and run locally
* Run `npx eslint --fix`
* Run `npx prettier --write`
* Followed PR quality checklist to ensure that from a functional point of view no changes were made to the code



## References

* https://github.com/aws-samples/pcluster-manager/pull/287

## PR Quality Checklist

- [ ] I added tests to new or existing code
- [ ] I removed hardcoded strings and used our `i18n` solution instead (see [here](https://github.com/aws-samples/pcluster-manager/pull/175/commits/fdc6b77987c87a26f51dbc8da5d371d95ef80601))
- [x] I checked that `npm run build` builds without any error
- [x] I checked that clusters are listed correctly
- [x] I checked that a new cluster can be created (config is produced and dry run passes)
- [x] I checked that login and logout work as expected

In order to increase the likelihood of your contribution being accepted, please make sure you have read both the [Contributing Guidelines](../CONTRIBUTING.md) and the [Project Guidelines](../PROJECT_GUIDELINES.md)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
